src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -246,16 +273,22 @@ public AzureAppConfigurationOptions UseFeatureFlags(Action<FeatureFlagOptions> c
                 var featureFlagFilter = featureFlagSelector.KeyFilter;
                 var labelFilter = featureFlagSelector.LabelFilter;
 
-                Select(featureFlagFilter, labelFilter);
+                ValidateSelectFilters(featureFlagFilter, labelFilter);
+
+                if (string.IsNullOrWhiteSpace(labelFilter))
+                {
+                    labelFilter = LabelFilter.Null;
```

Comment:
if the label filter is null,  we don't change `featureFlagSelector.LabelFilter`. Is that expected?

If no, then I don't think we need the variables `featureFlagFilter` and `labelFilter`

Reply:
I guess that's how it's behaved in the past since this was just calling `Select` before, so it wasn't changing `featureFlagSelector.LabelFilter`. It does seem like it makes more sense to just edit the `featureFlagSelector`, unless there's a reason I'm missing